packages/Button/src/LinkButton.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import Button from "./Button";
+import buttonStyles from "./Button.styles";
+
+const LinkPropTypes = {
+  children: PropTypes.node.isRequired,
+  kind: PropTypes.oneOf(Button.Kinds.ALL),
+  size: PropTypes.oneOf(ShirtSizes.MEDIUM),
+  href: PropTypes.string,
+  text: PropTypes.string,
```

Comment:
We should not need a `text` prop but just rely on `children` instead. Let me know if you have any questions

Reply:
done